courses/serializers.py

```diff
@@ -390,6 +388,56 @@ class Meta:
         ]
 
 
+class SimpleProgramSerializer(serializers.ModelSerializer):
```

Comment:
Does it make sense to update the existing ProgramSerializer?

Reply:
https://github.com/mitodl/mitxonline/pull/1708/commits/5e18fc739fb27ba420f76e75160854d56af5591b does that; there were other fields that the current frontend doesn't use that got pulled into `FullProgramSerializer` that inherits from `ProgramSerializer`.